ssl_proxy_vars.yml

```diff
@@ -0,0 +1,24 @@
+---
+nginx_sites:
+  ssl_reverse_proxy:
+    - listen 443 ssl
+    - ssl_certificate wildcard.tsuru.paas.alphagov.co.uk.site.chain.crt
+    - ssl_certificate_key wildcard.tsuru.paas.alphagov.co.uk.key
+    - server_name tsuru.paas.alphagov.co.uk
```

Comment:
It was moved as-is from the router (previously sslproxy). I suspect the `server_name` probably doesn't matter, because it's the only vhost listening on 443 so it will receive all requests regardless of `Host` header. We should make it factually correct though.